torch/csrc/jit/mobile/function.cpp

```diff
@@ -38,7 +38,9 @@ void Function::append_operator(const std::string& name,
   auto opname = code_->op_names_.back();
   // Add "_" prefix to work around the double registration both of jit/generated
   // and here. TODO: remove it when we have separate build for lite interpreter.
-  opname.name = "_" + opname.name;
+  if (opname.name != "aten::Int") {
```

Comment:
this is very strange. certainly not ok without a comment, probably not ok in general.

Reply:
Yeah it does look strange. It may not be possible to move all Aten ops to C10 in one shot, so the plan is to move them incrementally. I'll add a comment there. Hopefully this moving process will not last long so that the if statement with comment will be there temporarily.